geometry_msgs/PoseStamped pose
---
bool status